README.md

```diff
@@ -132,6 +134,7 @@ We use `react-router` [route definitions](https://github.com/reactjs/react-route
 )]
  - Incorporating server-side rendering (_very, very incomplete_) [[here](https://github.com/BigRoomStudios/strangeluv/compare/recipe-server-side
 )]
+ - Basic login recipe with account creation, authenticated routes and password reset (_requires an API with SES for password reset_) [[here](https://github.com/BigRoomStudios/strangeluv/compare/recipe-login)]
```

Comment:
I'd probably note that it's design to work with the user boilerplate (which is really imposing the SES requirement -- but that's not really a front end concern).

Reply:
oh good point, want me to link to the user boilerplate repo? are we still using matt's repo?